docs/platforms/Swift-Package-Manager.md

```diff
@@ -0,0 +1,645 @@
+                // privacy impact, and then uncomment these lines. For more information, see
+                // https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
+                // .process("PrivacyInfo.xcprivacy"),
+
+                // If you have other resources that need to be bundled with your plugin, refer to
+                // the following instructions to add them:
+                // https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package
+            ],
+            cSettings: [
+                .headerSearchPath("include/plugin_name_ios")
+            ]
+        )
+    ]
+)
+```
+
+* **If the plugin name contains `_`, the library name must be a `-` separated version of the plugin name.**
+5. If your plugin has a `PrivacyInfo.xcprivacy`, move it to `Sources/plugin_name_ios/PrivacyInfo.xcprivacy` and uncomment the resource in the Package.swift.
+```diff
+            resources: [
+                // If your plugin requires a privacy manifest, for example if it uses any required
+                // reason APIs, update the PrivacyInfo.xcprivacy file to describe your plugin's
+                // privacy impact, and then uncomment these lines. For more information, see
+                // https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
+-                // .process("PrivacyInfo.xcprivacy"),
++                .process("PrivacyInfo.xcprivacy"),
+
+                // If you have other resources that need to be bundled with your plugin, refer to
+                // the following instructions to add them:
+                // https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package
+            ],
+```
+6. Move any resource files from `ios/Assets` to `Sources/plugin_name_ios` (or a subdirectory). Then add them to your Package.swift if applicable. See https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package for more instructions.
+7. Move any public headers from `ios/Classes` to `Sources/plugin_name_ios/include/plugin_name_ios`
+    * If you're unsure what headers are public, check your `podspec` for `public_header_files`. If not found, that means all of your headers were public. You should consider whether or not you want all of your headers to be public.
+    * The `pluginClass` defined in your pubspec.yaml must be public and within this directory.
+8. Handling modulemap (skip this step if not using a custom modulemap)
+
+    If you're using a modulemap for CocoaPods to create a Test submodule, consider removing it for Swift Package Manager. Note that this will make all public headers available via the module.
+
+    To remove it for Swift Package Manager but keep it for CocoaPods, exclude the modulemap and umbrella header in the plugin's Package.swift. The example below assumes they are located within the `Sources/plugin_name_ios` directory.
```

Comment:
```suggestion
    To remove it for Swift Package Manager but keep it for CocoaPods, exclude the modulemap and umbrella header in the plugin's Package.swift. The example below assumes they are located within the `Sources/plugin_name_ios/include` directory.
```

Below in the podspec section, we recommend `s.public_header_files = 'plugin_name_ios/Sources/plugin_name_ios/include/**/*.h'`, so these files should probably be there

Reply:
Agreed, updated!